src/c++/perf_analyzer/genai-perf/genai_perf/wrapper.py

```diff
@@ -69,34 +69,31 @@ def build_cmd(args, extra_args):
             "random_seed",
         ]
 
-        if hasattr(args, "version") and args.version:
-            cmd = f"perf_analyzer --version"
-        else:
-            utils.remove_file(args.profile_export_file)
+        utils.remove_file(args.profile_export_file)
```

Comment:
Does this remove the profile export file even if someone is doing `genai-perf --version`?

Reply:
It does not. It exits immediately and doesn't get to this code